Replicator/Checkpoint.cc

```diff
@@ -91,7 +91,7 @@ namespace litecore { namespace repl {
                     C4SequenceNumber first = i->asInt();
                     C4SequenceNumber last = (++i)->asInt();
                     if (last >= first)
-                        _completed.add(first, last + 1);
+                        _completed.add(first, first + last);
```

Comment:
What is this change for?

Reply:
This relates to "CBL-4140: Inconsistent serialization/deserialization of Checkpoint::_completed"